src/components/About/Directors.module.css

```diff
@@ -0,0 +1,90 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
```

Comment:
Wont the above suggestion works the same way as this?

Reply:
Thank you, I'll make that change!